src/tests.zig

```diff
@@ -2249,3 +2253,68 @@ test "tagged userdata" {
     lua.pushInteger(13);
     try expectError(error.Fail, lua.userdataTag(-1));
 }
+
+fn vectorCtor(l: *Lua) i32 {
```

Comment:
I looked a little, but didn't see anything. Is there not a built-in way to build a vector in Luau by default?

Reply:
I was unable to construct vector values without registering the constructor myself.  I saw that the Rust mlua package was doing this too.  And well, Luau unit tests: https://github.com/luau-lang/luau/blob/f31232d301e9b738fca366a7924adad94ab18f27/tests/Conformance.test.cpp#L492

This is not the whole story though.  The vector constructor name can be passed as a Luau compile option (to luau_compile) so I assume that there's some fast path in the Luau VM related to creating vectors.  I'll investigate this more but I'm thinking this will be done in a separate PR.

Reply:
Interesting. I agree, that this can be looked into more later